src/handler.js

```diff
@@ -1,11 +1,24 @@
+class ProblemResponse extends Error {
+  constructor (code, detail, requestUri) {
+    super(detail)
+    this.statusCode = code
+    this.type = 'about:blank '
```

Comment:
You don't need to do this, I think

Reply:
It is just a small, reusable wrapper to separate our stuff from what api-gateway needs to have - to push errors to callers